packages/api/tests/lib/test-granules.js

```diff
@@ -1,12 +1,51 @@
 const test = require('ava');
 
+const { createBucket } = require('@cumulus/aws-client/S3');
```

Comment:
I believe all the changes in this file are now unnecessary and should be removed

Reply:
👍 just left one change to alpha-sort an import:

https://github.com/nasa/cumulus/pull/2057/commits/667e41a603d23b4fc2f553be40504132806f95d0